test/subdataframe.jl

```diff
@@ -124,4 +124,10 @@ module TestSubDataFrame
                                     y: Array{Float64}((5,)) [2.0, 3.0, 4.0, 5.0, 6.0]
                                   """
     end
+
+    @testset "deleterows!" begin
+        y = collect(1.0:10.0)
```

Comment:
We automatically collect ranges already.

Reply:
OK. I fixed all usages of `collect` also above.